.asf.yaml

```diff
@@ -39,3 +39,8 @@ github:
     squash: true
     merge: false
     rebase: false
+
+  protected_branches:
+    master:
+      required_pull_request_reviews:
```

Comment:
Do we need `require_code_owner_reviews: true` as well?
Ref: https://cwiki.apache.org/confluence/display/INFRA/git+-+.asf.yaml+features

Reply:
We don't have Code Owners defined in the repo though

Reply:
https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners